com.stansassets.scene-management/Editor/Visualizer/StateStackVisualizer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine.UIElements;
+
+namespace StansAssets.SceneManagement
```

Comment:
Please put all visualizer stuff under different namespace.
```suggestion
namespace StansAssets.SceneManagement.StackVisualizer
```